src/arrayMethodSort.js

```diff
@@ -5,7 +5,29 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
move default compareFunction to a variable
```suggestion
  [].__proto__.sort2 = function(compareFunction = compareByDefault) {
```